src/implementations/minecraft/mod.rs

```diff
@@ -43,21 +43,34 @@ use crate::util::{download_file, format_byte, format_byte_download, unzip_file,
 use self::players_manager::PlayersManager;
 use self::util::{get_jre_url, get_server_jar_url, read_properties_from_path};
 
-#[derive(Debug, Clone, TS, Serialize, Deserialize)]
+#[derive(Debug, Clone, TS, Serialize, Deserialize, PartialEq)]
+#[ts(export)]
+pub struct FabricLoaderVersion(String);
+#[derive(Debug, Clone, TS, Serialize, Deserialize, PartialEq)]
+#[ts(export)]
+pub struct FabricInstallerVersion(String);
+#[derive(Debug, Clone, TS, Serialize, Deserialize, PartialEq)]
+#[ts(export)]
+pub struct PaperBuildVersion(i64);
+#[derive(Debug, Clone, TS, Serialize, Deserialize, PartialEq)]
+#[ts(export)]
+pub struct ForgeBuildVersion(String);
```

Comment:
Probably a good idea to add conversion functions such as `From` and `AsRef` to add the ability to treat these new types as a string